x-pack/plugins/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -81,17 +87,29 @@ export const CaseViewActivity = ({
   showAlertDetails?: (alertId: string, index: string) => void;
   useFetchAlertData: UseFetchAlertData;
 }) => {
+  const [userActivityQueryParams, setUserActivityQueryParams] = useState<UserActivityParams>({
+    type: 'all',
+    sortOrder: 'asc',
+  });
+
   const { permissions } = useCasesContext();
+
   const { caseAssignmentAuthorized, pushToServiceAuthorized } = useCasesFeatures();
 
   const { data: caseConnectors, isLoading: isLoadingCaseConnectors } = useGetCaseConnectors(
     caseData.id
   );
 
   const { data: userActionsData, isLoading: isLoadingUserActions } = useFindCaseUserActions(
-    caseData.id
+    caseData.id,
+    {
+      ...userActivityQueryParams,
+    }
```

Comment:
nit: Probably it is not affecting React Query but I think we should pass the same instance (not spreading) of the object to the hook.